velox/common/caching/SsdFile.cpp

```diff
@@ -839,6 +857,28 @@ void SsdFile::initializeCheckpoint() {
   }
 }
 
+uint32_t SsdFile::checksumEntry(const AsyncDataCacheEntry& entry) {
+  // Always have 0 checksum when checksum is disabled.
+  if (!checksumEnabled_) {
+    return 0;
+  }
+  bits::Crc32 crc;
```

Comment:
Please check with @Yuhta if this is fast crc32 calculation.

Reply:
This should be very fast when each run is large enough